integration/integration_test.go

```diff
@@ -1,4 +1,4 @@
-//go:build integration || vm_integration || module_integration || k8s_integration
+//go:build integration || vm_integration || module_integration || k8s_integration || plugin_integration
```

Comment:
The VM, module, and K8s tests take longer to run and require a special setup. That's why they are separated. I think the plugin test can be part of the normal integration tests.

Reply:
updated in https://github.com/aquasecurity/trivy/pull/7299/commits/33cf9d5ee6faed00aa939cd33731c11ef2f8b738